ext/snowcrash/ext/markdown-parser/src/MarkdownParser.cc

```diff
@@ -57,10 +57,6 @@ void MarkdownParser::parse(const ByteBuffer& source, MarkdownNode& ast)
     m_source = NULL;
     m_sourceLength = 0;
     m_listBlockContext = false;
-
-#ifdef DEBUG
```

Comment:
Why is DEBUG removed?

Reply:
Unless we have an alternative, please don't remove it. It's pretty useful when debugging.

Reply:
I'm still convinced writing debug information to standard output out of nowhere is a bit too hacky. You can always log like this:

```
LOG(debug) << "Hello " << 42 << " or some other value";
```

This amounts to the same thing but can be dynamically turned on with a flag: `drafter -L` or `drafter --enable-log`. Also, it logs into the file `drafter.log` instead of polluting standard output. If needed, the `ENABLE_LOGGING` macro can be used to force logging while debugging.

Reply:
This shall be solved in a future PR, for now I'm reverting the change.